.github/workflows/pythonpackage.yaml

```diff
@@ -25,12 +25,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          pip install poetry
+          pipx install poetry==1.2.2
```

Comment:
Why doesn't this break on Python 3.6?

Reply:
Because, in switching to pipx, the tests now all run against Python 3.8.

https://github.com/fsfe/reuse-tool/actions/runs/3939665159/jobs/6739785792

This PR needs to be reverted.